### PR TITLE
more FAQ entries

### PR DESCRIPTION
**Pull request by Geal**

### CODE REVIEW

File: content/docs/help/faq.md [hunk 1]
```diff
@@ -16,5 +16,22 @@ top = false
...
 
+## How can biscuit be integrated with OAuth and OIDC?
+
+Biscuit is focused on the authorization and delegation side of API access, so it can be used as an OAuth access token. OAuth does not mandate a specific format for the access token.
+Refresh tokens work better when they are meant for a single use. In that case, a random string stored in the Authorization Server (in OAuth terms) database will fit the task.
+OIDC specifies that identity information is transmitted using JWT, so Biscuit will not be usable in their place.
+
+## Why Datalog?
+
+When searching for a suitable language to write authorization policies, it appeared that logic languages are generally a good fit. Other projects like OPA and Oso came to the same conclusion. Datalog is simple enough to be taught in a few minutes, and its implementation is straightforward, which makes it a good choice for safe authorization.
```
[the remark this answers is not in the document]
Comment (divarvel) [reply content/docs/help/faq.md:27]: why a custom language?: it can be embed in tokens and have the same behaviour on several language stacks
why a datalog variant?: being able to express rules and conditions is a good fit for authorization polices. Datalog has well-understood semantics and has been used in this domain for some time. It is also more flexible and more expressive than purely descriptive DSLs like what is available in zanzibar: biscuit aims at letting you choose the authorization model
why not an existing datalog? simple semantics, reduced dependency footprint, runtime constraints (memory used, time spent)

File: content/docs/help/faq.md [hunk 2]
```diff
@@ -16,5 +16,22 @@ top = false
...
+
+## Isn't JWT enough?
+
+JWT fits a specific use case: transmitting identity and authorization information from one central service to multiple less trusted services without tampering it. This works well for simple architectures like monoliths, but will soon force some tradeoffs in complexity or safety when used in today's microservices or federate architectures.
+It was also specified with a number of pitfalls that created a number of security incidents over the years.
+
+Biscuit builds upon the experience earned building authorization systems with JWT and other tools, takes steps to address their risks, and clearly target the larger systems we are now buiding.
```
Comment (woile) [inline content/docs/help/faq.md:34]:
```diff
- buiding
+ building
```

Comment (Geal) [reply content/docs/help/faq.md]: fixed in [d924ff8](https://github.com/biscuit-auth/website/pull/102/commits/d924ff8b6c78e3fbf09d810185fda61795d28b12)

File: content/docs/help/faq.md [hunk 3]
```diff
@@ -48,6 +48,26 @@ Of course biscuit is a recent piece of tech, which makes it harder to justify th
...
 
+## How can biscuit be integrated with OAuth and OIDC?
+
+Biscuit is focused on the authorization and delegation side of API access, so it can be used as an OAuth access token. OAuth does not mandate a specific format for the access token.
+Refresh tokens work better when they are meant for a single use. In that case, a random string stored in the Authorization Server (in OAuth terms) database will fit the task.
+OIDC specifies that identity information is transmitted using JWTs, so Biscuit will not be usable in their place.
+
+## Why Datalog?
+
+Biscuit needed a language that could be serialized in a compact format, and executed in the same way across different implementations in different languages. It also had to be fast to execute and have a limited impact on performance. And it had to be general enough, so it would not prescribe a specific authorization model.
```
Comment (divarvel) [inline content/docs/help/faq.md:59]:
```suggestion
Biscuit needed a language that could be serialized in a compact format, and executed in the same way across different implementations in different languages. It also had to be fast to execute and have a limited impact on memory use. Finally, it had to be general enough, so it would not prescribe a specific authorization model.
```

Comment (divarvel) [reply content/docs/help/faq.md:59]: as for performance, speed is mentionned just before, so it reads a bit strange to have both speed and performance mentioned separately.

**Review by divarvel — APPROVED** [review]
LGTM, with a minor suggestion

File: content/docs/help/faq.md [hunk 4]
```diff
@@ -48,6 +48,26 @@ Of course biscuit is a recent piece of tech, which makes it harder to justify th
 
 Consider unionizing to protect yourself from being unduly fired.
 
+## How can biscuit be integrated with OAuth and OIDC?
```
Comment (woile) [inline content/docs/help/faq.md:51]: I would rephrase this making it clear that there are 3 types of tokens and biscuit fits only one. I remember learning oauth and mixing all the types of token together, specially the identity and access tokens. Something like:

You can find 3 different types of tokens on Oauth and oidc, biscuit only fit in one:
- access token: Biscuit is focused on the authorization and delegation side of API access, so it can be used as an OAuth access token. OAuth does not mandate a specific format for the access token.
- refresh token: ...
- identity token: oidc mandates the identity token should be a jwt...

Comment (divarvel) [reply content/docs/help/faq.md:51]: there's an open issue for a more in-depth discussion of oauth / oidc #14
